.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Take a look at the project; you'll notice that this isn't actually the style used throughout.